tests/integration/fixture/gloss_widget.html

```diff
@@ -26,8 +26,8 @@
             </table>
         </div>
         <div class="wikibase-lexeme-sense-glosses-controls">
-            <button type="button" class="wikibase-lexeme-sense-glosses-control" v-if="!inEditMode" v-on:click="edit" :disabled="isSaving">{{message('wikibase-edit')}}</button>
-            <button type="button" class="wikibase-lexeme-sense-glosses-control" v-if="inEditMode" v-on:click="save" :disabled="isSaving">{{message('wikibase-save')}}</button>
+            <button type="button" class="wikibase-lexeme-sense-glosses-control" v-if="!inEditMode" v-on:click="edit" :disabled="isSaving">{{ message('wikibase-edit') }}</button>
+            <button type="button" class="wikibase-lexeme-sense-glosses-control" v-if="inEditMode" v-on:click="save" :disabled="isSaving">{{message( 'wikibase-save' )}}</button>
```

Comment:
I don’t think we should test the spacing here – we probably won’t use it like that (at least not inconsistently) in WikibaseLexeme itself, and I’d prefer to limit differences between these fixtures and the real WikibaseLexeme templates. (Though they’ve apparently already diverged quite a bit 😕) The unit test below should be enough IMHO.

Reply:
Done.